### ISSUE TITLE
LRS Dashboard Mockups

### ISSUE THREAD

**Issue by pavel-ganapolsky** [bcgov/nr-epd-digital-services#266]
**Describe the task**
Creation of LRS Dashboard mockups for external, staff and manager users.

**Acceptance Criteria**
- [ ] Figma file containing mockups
- [ ] Mockups peer reviewed by team and approved by Haley

**UX Definition of Done**
- [ ] User stories reflected in mockups

**Comment by acoard-aot** [comment]
These designs need to include "Packaging/Bundling" before we can implement them.  Maybe we should create a separate task for that and mark it as blocking #113? Any thoughts?
